config/cesm/config_grids.xml

```diff
@@ -1233,41 +1233,40 @@
       <nx>288</nx>  <ny>192</ny>
       <file grid="atm|lnd" mask="gx1v6">domain.lnd.fv0.9x1.25_gx1v6.090309.nc</file>
       <file grid="ocnice"  mask="gx1v6">domain.ocn.0.9x1.25_gx1v6_090403.nc</file>
+
       <file grid="atm|lnd" mask="gx1v7">domain.lnd.fv0.9x1.25_gx1v7.151020.nc</file>
       <file grid="ocnice"  mask="gx1v7">domain.ocn.fv0.9x1.25_gx1v7.151020.nc</file>
-<!--  <file grid="atm|lnd" mask="null">/glade/u/home/benedict/ys/datain/domain.aqua.fv0.9x1.25.nc</file>
-      <file grid="ocnice"  mask="null">/glade/u/home/benedict/ys/datain/domain.aqua.fv0.9x1.25.nc</file> -->
-      <file grid="atm|lnd" mask="null">domain.aqua.fv0.9x1.25.nc</file>
-      <file grid="ocnice"  mask="null">domain.aqua.fv0.9x1.25.nc</file>
+
       <file grid="atm|lnd" mask="tnx1v1">domain.lnd.fv0.9x1.25_tnx1v1.150512.nc</file>
       <file grid="ocnice"  mask="tnx1v1">domain.ocn.0.9x1.25_tnx1v1.150512.nc</file>
-      <file grid="atm|lnd" mask="tnx1v3">domain.lnd.fv0.9x1.25_tnx1v3.170621.nc</file>
-      <file grid="ocnice"  mask="tnx1v3">domain.ocn.tnx1v3.170621.nc</file>
+
       <file grid="atm|lnd" mask="tnx1v4">domain.lnd.fv0.9x1.25_tnx1v4.170609.nc</file>
-      <file grid="ocnice"  mask="tnx1v4">domain.ocn.tnx1v4.170606.nc</file>
+      <file grid="ocnice"  mask="tnx1v4">domain.ocn.fv0.9x1.25_tnx1v4.170609_djlo.nc</file>
+
       <file grid="atm|lnd" mask="tnx0.25v1">domain.lnd.fv0.9x1.25_tnx0.25v1.140618.nc</file>
-      <file grid="atm|lnd" mask="tnx0.25v3">domain.lnd.fv0.9x1.25_tnx0.25v3.170629.nc</file>
-      <file grid="ocnice"  mask="tnx0.25v3">domain.ocn.tnx0.25v3.170629.nc</file>
-      <file grid="atm|lnd" mask="tnx0.25v4">domain.lnd.fv0.9x1.25_tnx0.25v4.170629.nc</file>
-      <file grid="ocnice"  mask="tnx0.25v4">domain.ocn.tnx0.25v4.170629.nc</file>
-      <file grid="atm|lnd" mask="tnx0.125v4">domain.lnd.fv0.9x1.25_tnx0.125v4.200830.nc</file>
-      <file grid="ocnice"  mask="tnx0.125v4">domain.ocn.tnx0.125v4.200725.nc</file>
-      <!--file grid="ocnice" mask="tnx0.25v1">domain.ocn.0.9x1.25_tnx0.25v1.140618.nc</file-->
+      <file grid="ocnice"  mask="tnx0.25v1">domain.ocn.0.9x1.25_tnx0.25v1.140618.nc</file>
+
+      <!-- <file grid="atm|lnd" mask="tnx0.25v3">domain.lnd.fv0.9x1.25_tnx0.25v3.170629.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.25v3"></file> -->
+
+      <!-- <file grid="atm|lnd" mask="tnx0.25v4">domain.lnd.fv0.9x1.25_tnx0.25v4.170629.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.25v4"></file> -->
+
+      <!-- <file grid="atm|lnd" mask="tnx0.125v4">domain.lnd.fv0.9x1.25_tnx0.125v4.200830.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.1.25v4"></file> -->
```

Comment:
Why are these commented out instead of being removed?

Reply:
> @gold2718 - I wanted to make it explicit that there is no available ocean domain file for f09. Previously a pointer to the tnx1v4 domain file, etc was used - and this is incorrect for an ocean grid that is f09.

1. A comment would be a lot more helpful (I did not know why they were commented out)
2. Are there no appropriate files in the NCAR svn inputdata repository?